### PR TITLE
Update cheatsheet with new section on creating yamls

### PR DESCRIPTION
**Pull request by elineex**

### CODE REVIEW

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -447,3 +447,95 @@ Verbosity | Description
 * Also read [kubectl Usage Conventions](/docs/reference/kubectl/conventions/) to understand how to use kubectl in reusable scripts.
 
 * See more community [kubectl cheatsheets](https://github.com/dennyzhang/cheatsheet-kubernetes-A4).
+
+
+# Fast makeing yamls
```
Comment (reylejano) [inline content/en/docs/reference/kubectl/cheatsheet.md:452]:
```suggestion
# Creating yaml manifests with kubectl
```

**Comment by reylejano** [comment]
/retitle Update cheatsheet with new section on creating yamls
 
I suggest adding a comment on what each command does like what kind of yaml manifest does the command create
Some examples from the Cheatsheet

> kubectl get services                          # List all services in the namespace
> kubectl get pods --all-namespaces             # List all pods in all namespaces
>

**Review by elineex — COMMENTED** [review]
Add comments

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 2]
```diff
@@ -447,3 +447,122 @@ Verbosity | Description
...
+
+# Creating yaml manifests with kubectl
+
+# Creating deployment my-dep in yaml file foo2.yaml and listen on port 5710
+
+kubectl create deployment my-dep --image=busybox --port=5701 --dry-run=none -o yaml >foo2.yaml
+
+# Creating secret for docker-registry
+
+kubectl create secret docker-registry foo --dry-run="none" --docker-username=foo --docker-password=foo --docker-server=127.0.0.1 -o yaml >foo3.yaml
```
Comment (Sea-n) [inline content/en/docs/reference/kubectl/cheatsheet.md:460]:
````suggestion
## Creating yaml manifests with kubectl
```bash
# Creating deployment my-dep in yaml file foo2.yaml and listen on port 5710
kubectl create deployment my-dep --image=busybox --port=5701 --dry-run=none -o yaml >foo2.yaml

# Creating secret for docker-registry
kubectl create secret docker-registry foo --dry-run="none" --docker-username=foo --docker-password=foo --docker-server=127.0.0.1 -o yaml >foo3.yaml
```
````

**Review by Sea-n — COMMENTED** [review]
I think you are not intended to create so many h1 header, maybe this format would be better.

**Comment by divya-mohan0209** [comment]
Hey @elineex , please let us know if you'd be willing to incorporate suggestions by @Sea-n ?

**Comment by Sea-n** [comment]
Hey @sean,

Thanks for purposing this PR. But since you didn't respond for a while, we have to close this PR now.
/close

Feel free to `/reopen` when you are ready to move forward.

**Pull request closed without merging**